### PR TITLE
Create installation flow webpages

### LINKED ISSUE #58: Create GithubApp to avoid needing the GITHUB_TOKEN from the action.
- Add herald GitHub app to avoid asking for the secret in the PR configuration. (leave it optional)
- Add documentation to install the GitHub app.

### PR DESCRIPTION
**Pull request by cyamonide**
**Issue Reference**: #83, closes #58 

## Description

This PR adds the installation flow in the form of a GitHub-hosted site, at `gagoar.github.io/use-herald-action`

## How Has This Been Tested?

Manually, using my own domain `cyamonide.github.io/use-herald-action`.

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/create.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+
+<head>
+  <title>UHA - Creating the App...</title>
```
Comment (gagoar) [inline docs/create.html:4]:
```suggestion
  <title>Use Herlad Action - Creating the App...</title>
```

File: docs/index.html [hunk 2]
```diff
@@ -1,9 +1,198 @@
 <!DOCTYPE html>
 
 <head>
-  <title>This is a test</title>
+  <title>UHA - Install the App</title>
```
Comment (gagoar) [inline docs/index.html:4]:
```suggestion
  <title>Use Herald Action - Install the App</title>
```

File: docs/token.html [hunk 3]
```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
+
+<head>
+  <title>UHA - Completing Handshake...</title>
```
Comment (gagoar) [inline docs/token.html:4]:
```suggestion
  <title>Use Herald Action - Completing Handshake...</title>
```

File: docs/token.html [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+    document.getElementById('header-text').innerHTML = 'Something went wrong...<br />Give that another go, will ya?';
+
+    document.title = 'UHA - Validation failed';
+  };
+
+  document.getElementById('steps-parent').style.display = 'none';
+  document.getElementById('steps-parent').style.opacity = '0';
+
+  // Get URL parameters
+  const { code } = getParams(window.location.href);
```
Comment (gagoar) [inline docs/token.html:256]: I haven't done anything in the browser for so long, that I didn't know you could do Destructuring assignment.

Comment (cyamonide) [reply docs/token.html:256]: I was surprised too - I had to try it, and expected it to fail, but it didn't!

**Review by gagoar — APPROVED** [review]
Looks amazing.  🎉 
Thank you for tackling this big problem. 🤯 

Some comments about the title, and 2  additions:
-  can you add a step so we can share the icon? ppl might wanna give it a logo. 
-  some documentation feels is needed to explain the permissions issue, send them to this Wizzard and then know how to use that in the workflow.

**Pull request merged**